main.go

```diff
@@ -149,11 +148,6 @@ func main() {
 		log.Error(err, "")
 		os.Exit(1)
 	}
-	log.Info("Creating the engine CRDs")
-	if err := engine.Ensure(crdUpdater); err != nil {
-		log.Error(err, "")
-		os.Exit(1)
-	}
 	log.Info("Creating the Lighthouse CRDs")
```

Comment:
We should ideally install these only if LH, but now globalnet also uses these CRDs.

Reply:
OK, so I need to include the Globalnet componentisation in here or hack something...